lib/NativeCode/DynamicLibraryLoaderHelper/NativeRender/include/logging.h

```diff
@@ -38,6 +38,13 @@ namespace pew::eos::logging
 {
     typedef void (*log_flush_function_t)(const char* str);
 
+    /**
+     * \brief Allows an option to be set that will mirror the log messages to
```

Comment:
I don't recognize this commenting syntax, but judging by below, maybe this should be `@brief`?

Reply:
Doxygen comment formatting accepts either option. This is the format that Visual Studio will auto-insert when you type `/**` and hit enter (if the feature is turned on). Visual Studio and other IDEs will also correctly interpret these into intellisense.